lib/pxconfig.js

```diff
@@ -307,43 +304,49 @@ function pxDefaultConfig(PX_INTERNAL) {
 }
 
 const configSchemaMapper = {
-    px_enable_module: 'enableModule',
-    px_app_id: 'pxAppId',
-    px_cookie_secret: 'cookieSecretKey',
-    px_auth_token: 'authToken',
-    px_proxy_url: 'proxy',
-    px_sync_request_timeout_ms: 'apiTimeoutMS',
-    px_custom_request_handler: 'customRequestHandler',
-    px_get_user_ip: 'getUserIp',
-    px_blocking_score: 'blockingScore',
-    px_ip_headers: 'ipHeaders',
-    px_send_async_activities: 'sendPageActivities',
-    px_sensitive_headers: 'sensitiveHeaders',
-    px_debug_mode: 'debugMode',
-    px_max_buffer_length: 'maxBufferLength',
-    px_js_ref: 'jsRef',
-    px_css_ref: 'cssRef',
-    px_custom_logo: 'customLogo',
-    px_filter_by_http_method: 'filterByMethod',
-    px_filter_by_ip: 'filterByIP',
-    px_filter_by_user_agent: 'filterByUserAgent',
-    px_sensitive_routes: 'sensitiveRoutes',
-    px_enforced_routes: 'enforcedRoutes',
-    px_whitelist_uri_full: 'whitelistRoutes',
-    px_dynamic_configurations: 'dynamicConfigurations',
-    px_module_mode: 'moduleMode',
-    px_first_party_enabled: 'firstPartyEnabled',
-    px_additional_activity_handler: 'additionalActivityHandler',
-    px_enrich_custom_parameters: 'enrichCustomParameters',
-    px_test_mode: 'testingMode',
-    px_whitelist_extensions: 'whitelistExt',
-    px_bypass_monitor_header: 'bypassMonitorHeader',
-    px_advanced_blocking_response: 'advancedBlockingResponse',
-    px_telemetry_command: 'telemetryCommandHeader',
-    px_pxhd_secure: 'pxhdSecure',
-    px_backend_url: 'backendUrl',
-    px_login_credentials_extraction_enabled: 'enableLoginCredsExtraction',
-    px_login_credentials_extraction: 'loginCredsExtraction',
+    px_module_enabled: 'px_module_enabled',
+    px_app_id: 'px_app_id',
+    px_cookie_secret: 'px_cookie_secret',
+    px_auth_token: 'px_auth_token',
+    px_proxy_url: 'px_proxy_url',
+    px_s2s_timeout: 'px_s2s_timeout',
+    px_batch_activities_timeout_ms: 'px_batch_activities_timeout_ms',
+    px_custom_request_handler: 'px_custom_request_handler',
+    px_extract_user_ip: 'px_extract_user_ip',
+    px_blocking_score: 'px_blocking_score',
+    px_ip_headers: 'px_ip_headers',
+    px_send_async_activities_enabled: 'px_send_async_activities_enabled',
+    px_sensitive_headers: 'px_sensitive_headers',
+    px_logger_severity: 'px_logger_severity',
+    px_max_activity_batch_size: 'px_max_activity_batch_size',
+    px_js_ref: 'px_js_ref',
```

Comment:
really? haha
I think we don't need it anymore, there is no need to map anymore, no?

Reply:
Yeah, I'll get rid of it.